examples/erc20/tests/erc20.rs

```diff
@@ -32,6 +49,29 @@ async fn deploy(
     e2e::deploy(rpc_url, private_key, Some(args)).await
 }
 
+fn prepare_typed_data_hash(domain_separator: B256, struct_hash: B256) -> B256 {
```

Comment:
`prepare` is generally an ambiguous word, cause it doesn't really say much about what the preparation entails, so it's better to avoid it.

I would just rename it to `to_typed_data_hash`.